catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -173,6 +174,23 @@ public void init() {
         GSON::toJson);
 
     put(
+        "/forms/query/:id",
```

Comment:
It's a little awkward that we are taking in the id and then not using it. Maybe we can refactor the `doCreateOrUpdate` function call to two separate functions where one takes in an id.